packages/forms/src/Components/Section.php

```diff
@@ -10,11 +10,16 @@
 class Section extends Component implements Contracts\CanConcealComponents, Contracts\CanEntangleWithSingularRelationships
 {
     use Concerns\CanBeCollapsed;
+    use Concerns\CanBeCompacted;
     use Concerns\EntanglesStateWithSingularRelationship;
     use HasExtraAlpineAttributes;
 
     protected string $view = 'forms::components.section';
 
+    public const REGULAR = 'forms::components.section';
+
+    public const SMALL = 'forms::components.section-small';
```

Comment:
I think we'd rather have abstracted methods like `compact()` instead of having to pass the const to the `view()` method. Take a look at actions for example, with their `button()` and `link()` methods.

Reply:
Yes exactly

Reply:
Sorry: I forgot to clean up those contants.
abstract methods are implemented in the `CanBeCompacted` trait